client/core/core.go

```diff
@@ -4673,7 +4673,7 @@ func (c *Core) dbTrackers(dc *dexConnection) (map[order.OrderID]*trackedTrade, e
 		for _, dbMatch := range dbMatches {
 			// Only trade matches are added to the matches map. Detect and skip
 			// cancel order matches, which have an empty Address field.
-			if dbMatch.Address == "" {
+			if dbMatch.Address == "" { // only correct for maker's cancel match
```

Comment:
This is a sorta redundant check anyway since `MatchesForOrder` a few lines up should screen them out, but if it fails to do so, then this check is effective for screening out this trade order's cancel match.